server/api/controllers/forms/get-form.js

```diff
@@ -9,17 +9,33 @@ module.exports = {
       type: "string",
       description: "Id of form",
     },
+    isLoggedIn: {
+      type: "boolean",
+      description: "whether the user is logged in or not",
+      required: true,
+    },
   },
 
   exits: {},
 
-  fn: async function ({ id }, exits) {
+  fn: async function ({ id, isLoggedIn }, exits) {
     try {
-      const data = await Form.find({
+      let searchParams = {
         _id: id,
         isPublished: true,
         isDeleted: false,
-      });
+      };
```

Comment:
```
searchParams = {
  _id: id,
  isPublished = true,
  isDeleted = false,
  requireLogin = !isLoggedIn
};
```

Reply:
no i realized can't do this. if we do this then it would only return forms where `requireLogin = !isLoggedIn`

if `isLoggedIn` is `true` then we want to get all forms whether their `requireLogin` is `true` or `false`. But when `isLoggedIn` is `false` then we should only return the forms where `requireLogin` is `false`. Unless I'm not getting how the params search works?

Reply:
Ah ok I get what you mean

Reply:
Fine to leave it as it is now. Wondering if null works?

```
requireLogin: !isLoggedIn ? isLoggedIn : null
```